br/pkg/lightning/backend/local/local.go

```diff
@@ -1776,6 +1788,12 @@ func (local *local) deleteDuplicateRows(ctx context.Context, logger *log.Task, h
 	// (if the number of duplicates is small this should fit entirely in memory)
 	// (Txn's MemBuf's bufferSizeLimit is currently infinity)
 	for _, handleRow := range handleRows {
+		// Skip the row key if it's not in the table.
+		// This can happen if the table has been recreated or truncated,
+		// and the duplicate key is from the old table.
+		if !keyInTable(handleRow[0]) {
```

Comment:
If it's not urgent to fix, can we add more information to conflict_error_v1 and other coordination tables like taskID, taskStartTime to filter the outdated duplicated records?

Reply:
This fix will be cherry-picked to the released version like v6.5. It's better not to change the table schema. 
Although lightning records self-task ID, it's difficult to determine whether the records are inserted by other lightning instances during parallel import or just by a previous import.

Reply:
Seems we also can't easily distinguish outdated conflict_error_v1 records in precheck, because it may belong to a parellel import instance?

We rely on the specific way to clean up data too much. If the user choose batch delete to clean the table for some reason (though it's almost impossible), this check will not work. But I also can't find another fix so almost lgtm.

Reply:
> Seems we also can't easily distinguish outdated conflict_error_v1 records in precheck, because it may belong to a parellel import instance?

Yes.

> If the user choose batch delete to clean the table for some reason (though it's almost impossible), this check will not work.

Currently, duplicate detection doesn't work with increment import well. That is to say if users perform CRUD operations on the target table. Lightning's duplicate detection even may fail or return wrong results. For this scenario, there is still much work to do.